cpp/models/sde_seir2v/README.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Please write a meaningful README. For reference, you can look at the ode_secir/ode_secirvvs READMEs (but, please, do not copy them).

Specifically explain the compartments with mixed naming/the possible infection paths. For the parameters, you can add a table similar to the other model READMEs with name and description, optionally a mathematical variable if you refer to a paper (if so, cite it).